src/reusecore/PageHeader/index.js

```diff
@@ -46,7 +46,7 @@ const PageHeader = ({ category, title, img, feedlink, subtitle,  author, thumbna
     <PageHeaderWrapper>
       <div className="page-header">
         { thumbnail && <div className="feature-image">
-          <Image {...thumbnail} imgStyle={{ objectFit: "contain"}} alt={title}/>
+          <Image {...thumbnail} imgStyle={{ objectFit: "contain",  display: "block",marginLeft: "auto", marginRight: "auto"}} alt={title}/>
```

Comment:
Same as above

Reply:
In this file what changes do I need to do

Reply:
I mean as asked in the comment below i.e, move the changes to the imported CSS file.